backend/config/lightning.go

```diff
@@ -23,7 +23,7 @@ import (
 // LightningAccountConfig is the configuration of a single Lightning account.
 type LightningAccountConfig struct {
 	// Mnemonic is the wallet node generated from the device entropy.
-	Mnemonic string `json:"mnemonic"`
+	Mnemonic string `json:"-"`
```

Comment:
With this you delete the mnemonic also from the config, and the user can't spend anymore :cold_sweat: You even changed the test above to reflect this :sweat_smile:

Reply:
hmm, I'll look into it, thanks.